### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by codingwithdani**

### CODE REVIEW

**Comment by paulusrex** [comment]
Has marcado master en vez de develop

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Hello world
+
+## Intro
```
Comment (paulusrex) [inline README.md]: @codingwithdani Introduction better

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+TODO explain project
```
Comment (paulusrex) [inline README.md]: @codingwithdani Fill info instead of TODOs

File: README.md (hunk 1 above)
Comment (codingwithdani) [inline README.md]: @paulusrex intro, no "introduction" yes

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+##Functional Description
+
+TODO describe functionality
```
Comment (paulusrex) [inline README.md]: @codingwithdani Fill info instead of TODOs

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+## Technical description
+
+TODO describe specs
```
Comment (paulusrex) [inline README.md]: @codingwithdani Fill info instead of TODOs

File: README.md (hunk 3 above)
Comment (codingwithdani) [inline README.md]: You should finish this TODO @paulusrex

File: README.md (hunk 1 above)
Comment (paulusrex) [reply README.md]: I think that intro is fine @codingwithdani

**Pull request merged**